app/stable_appconfig.go

```diff
@@ -0,0 +1,34 @@
+// +build stable
+
+package app
+
+import (
+	"github.com/cosmos/cosmos-sdk/types/module"
+)
+
+func setCustomModuleBasics() []module.AppModuleBasic {
+	return []module.AppModuleBasic{}
+}
```

Comment:
basically in `stable` version, we'll not have any experimental modules integrated.